### PR TITLE
KANBAN-230

### PR DESCRIPTION
**Pull request by christabone**
- Adds blackbox http / endpoint monitoring.
- Enables sending warnings via AWS SNS service.

### CODE REVIEW

File: hosts [hunk 1]
```diff
@@ -18,3 +18,6 @@ localhost
 
 [buildserver]
 54.89.31.80
+
+[launched]
+34.236.154.139
```
Comment (oblodgett) [inline hosts:23]: Umm.. do you want to commit [launched] that is a dynamically generated list.

File: templates/prometheus_production.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    static_configs:
+      - targets: ['{{ NODEEXPORTER_SERVER_NAME }}:9100']
+
+  - job_name: 'blackbox'
+    metrics_path: /probe
+    params:
+      module: [http_2xx]
+    static_configs:
+      - targets:
+        - http://build.alliancegenome.org/api/search_autocomplete?q=p
```
Comment (oblodgett) [inline templates/prometheus_production.yml.j2:45]: production or build?

File: templates/prometheus.yml.j2 [hunk 3]
```diff
@@ -34,4 +34,19 @@ scrape_configs:
...
\ No newline at end of file
+      - targets: ['{{ NODEEXPORTER_SERVER_NAME }}:9100']
+
+  - job_name: 'blackbox'
+    metrics_path: /probe
+    params:
+        module: [http_2xx]
+    static_configs:
+        - targets:
+            - https://www.alliancegenome.org/api/search_autocomplete?q=p
```
Comment (oblodgett) [inline templates/prometheus.yml.j2:45]: Also its not clear which container is getting restarted when this link goes down.

File: templates/prometheus_production.yml.j2 (hunk 2 above)
Comment (christabone) [reply templates/prometheus_production.yml.j2:45]: Good catch, fixed.

File: hosts (hunk 1 above)
Comment (christabone) [reply hosts:23]: Whoops, fixed.

File: templates/prometheus.yml.j2 (hunk 3 above)
Comment (christabone) [reply templates/prometheus.yml.j2:45]: The AWS Lambda script is in the "agr_monitoring" repository. This just triggers an SNS message sent to AWS Lambda.

Comment (oblodgett) [reply templates/prometheus.yml.j2:45]: Ahh ok.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**